src/coreclr/vm/jitinterface.cpp

```diff
@@ -3839,6 +3839,57 @@ bool CEEInfo::isValueClass(CORINFO_CLASS_HANDLE clsHnd)
     return ret;
 }
 
+bool CEEInfo::isBitwiseEquatable(CORINFO_CLASS_HANDLE cls)
+{
+    CONTRACTL {
+        NOTHROW;
+        GC_NOTRIGGER;
+        MODE_PREEMPTIVE;
+    } CONTRACTL_END;
+
+    bool ret = false;
+
+    JIT_TO_EE_TRANSITION_LEAF();
```

Comment:
```suggestion
    JIT_TO_EE_TRANSITION();
```
This cannot be lead. It calls methods that can throw